itkwidgets/_type_aliases.py

```diff
@@ -15,21 +15,21 @@
 Style = Dict[str, str]
 
 Image = Union[np.ndarray, itkwasm.Image, zarr.Group]
-Point_Sets = Union[np.ndarray, itkwasm.PointSet, zarr.Group]
+Point_Set = Union[np.ndarray, itkwasm.PointSet, zarr.Group]
```

Comment:
unrelated to your changes here, but can we please fix regardless: `Point_Set` -> `PointSet` :pray:

Reply:
Fixed!